### PR TITLE
Add scams from ChainPatrol

### PR DESCRIPTION
**Pull request by NikitaVr**
## Scam links
- [arbappitrum.com](https://arbappitrum.com)
- [bridge-arbiltum.com](https://bridge-arbiltum.com)

## Description

Suspicious activity detected on these domains

## Screenshots

---

## Scam links
- [arb-bridge.online](https://arb-bridge.online)

## Description

This website is showing on some browsers when searching for "arbitrum bridge". It looks kind of similar to arbitrum's website, but it's actually a scam.

## Screenshots

---

## Scam links
- [register-argentwallet.com](https://register-argentwallet.com)

## Screenshots

---

## Scam links
- [https://zetachain.bounty-web3.network/](https://https://zetachain.bounty-web3.network/)

## Description

reported by discord user kinyoubi#0, Discord ID: 924675603611934850, from Server: 858516330432299008

## Screenshots

---

## Scam links
- [zksclaim-dashboard.com](https://zksclaim-dashboard.com)

## Description

Linked to from this fake Twitter profile: https://twitter.com/zk_syuc

## Screenshots

---

## Scam links
- [airdrop-taikos.xyz](https://airdrop-taikos.xyz)

## Screenshots

---

## Scam links
- [claim-mantle.com](https://claim-mantle.com)

## Description

High quality phishing scam

## Screenshots

---

## Scam links
- [lido-claim.net](https://lido-claim.net)

## Screenshots

---

## Scam links
- [defi-lido.fi](https://defi-lido.fi)

## Screenshots

---

## Scam links
- [lidoevent.com](https://lidoevent.com)
- [lidpool.fi](https://lidpool.fi)

## Description

Phishing URL

## Screenshots

---

## Scam links
- [web3-arbitrum.com](https://web3-arbitrum.com)

## Screenshots

---

## Scam links
- [airdrop.layezrero.network](https://airdrop.layezrero.network)

## Description

Phishing URL

## Screenshots

---

## Scam links
- [lineabuild-airdrops.com](https://lineabuild-airdrops.com)

## Screenshots

---

## Scam links
- [lineastacks.com](https://lineastacks.com)

## Description

reported by discord user chinzilla24#0

## Screenshots

---

## Scam links
- [linea-eth.axelar.pro](https://linea-eth.axelar.pro)

## Screenshots

---

## Scam links
- [https://lineabuild.netlify.app](https://https://lineabuild.netlify.app)

## Description

reported by discord user chinzilla24#0

## Screenshots

---

## Scam links
- [https://lineasbuild.com](https://https://lineasbuild.com)

## Description

reported by discord user chinzilla24#0

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -513,14 +513,31 @@
     "launchpad.ethereum.org"
   ],
   "blacklist": [
+    "https://lineabuild.netlify.app",
```
Comment (umar-ahmed) [inline src/config.json:516]: remove protocol

File: src/config.json [hunk 2]
```diff
@@ -513,14 +513,32 @@
     "launchpad.ethereum.org"
   ],
   "blacklist": [
+    "https://lineasbuild.com",
+    "https://lineabuild.netlify.app",
```
Comment (umar-ahmed) [inline src/config.json]:
```suggestion
    "lineasbuild.com",
    "lineabuild.netlify.app",
```

**Comment by umar-ahmed** [comment]
@AlexHerman1 I think the dedupe, while it fixed the build issue, might have introduce a merge conflict. Is it safe for me to do an interactive rebase and just omit that line where we added the zetachain URL? I think that might fix it

**Comment by AlexHerman1** [comment]
@umar-ahmed  should be good now

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

**Review by deshvin — APPROVED** [review]

**Comment by umar-ahmed** [comment]
> @umar-ahmed  should be good now

Thanks 🙏!

**Pull request merged**